R/funs-date.R

```diff
@@ -78,11 +78,19 @@ pl_second_lubridate <- function(x, ...) {
 }
 
 # Date-time helpers --------------------------------------
+# TODO: simplify when $dt$date() is in r-polars
+pl_date_lubridate <- function(x, ...) {
+  check_empty_dots(...)
 
-# TODO: implement when $dt$date() is in r-polars
-# pl_date_lubridate <- function(x) {
-#   x$dt$date()
-# }
+  if (is.na(x)) {
+    return(NA)
+  }
```

Comment:
I don't think this is necessary. If a value is NA (`null` in polars) then the operations below will also return NA. Did you get a case where this was needed?